### ISSUE TITLE
How to use it?

### ISSUE THREAD

**Issue by ChoiiYill** [3DOM-FBK/COLMAP_SLAM#6]
I've take 78 pictures, when I use it, the prompts come: "No more images available."
It could only handle two pictures.
![QQ图片20240113114138](https://github.com/3DOM-FBK/COLMAP_SLAM/assets/59330123/4f7d26a1-edbf-4665-b2c4-d63315b3b4e4)

**Comment by lcmrl** [comment]
Hi, did you manage to run it?

**Comment by ChoiiYill** [comment]
> Hi, did you manage to run it?

Yes, I've downloaded the dataset, and run dataset Machine Hall 02, but failed.
Here's some runtime prompts:

2024-01-15 18:35:41,603 - INFO - pointer 68 c 81
2024-01-15 18:35:41,671 - INFO - Pydegensac found 118/141 inliers
2024-01-15 18:35:41,671 - INFO - median_match_dist: 79.83
camera 0
2024-01-15 18:35:41,672 - INFO - Frame accepted. New_keyframe image_id: 149
2024-01-15 18:35:41,695 - INFO - [Keyframe Selection] | features extraction=0.049, matching=0.019, innovation check=0.001,
2024-01-15 18:35:41,696 - INFO - DYNAMIC MATCHING WINDOW: 1
2024-01-15 18:35:41,696 - INFO - Initialize an empty database
2024-01-15 18:35:41,696 - INFO - Feature extraction
2024-01-15 18:35:43,174 - INFO - Sequential matcher
2024-01-15 18:35:44,493 - INFO - MAPPER
2024-01-15 18:35:46,180 - INFO - Convert model from binary to txt
2024-01-15 18:35:46,323 - INFO - [COLMAP] | DATABASE INITIALIZATION=0.000, FEATURE EXTRACTION=1.478, SEQUENTIAL MATCHER=1.319, MAPPER=1.687, MODEL CONVERSION=0.143,
True
keyframe 1403636858651666432.jpg
Traceback (most recent call last):
  File "E:\Coding_Packages\COLMAP_SLAM\main.py", line 601, in <module>
    if k.keyframe_name ==  None:
AttributeError: 'NoneType' object has no attribute 'keyframe_name'

**Comment by lcmrl** [comment]
In theory should be a bug that rarely happen. I work on it. Let me know if instead it happens every time you try to run it
